user/deployment/codedeploy.md

```diff
@@ -17,6 +17,7 @@ For a minimal configuration with S3, add the following to your `.travis.yml`:
         secret_access_key: "YOUR AWS SECRET KEY"
         bucket: "S3 Bucket"
         key: latest/MyApp.zip
+        bundle_type: zip
```

Comment:
`bundle_type` is detected, so it should not appear in the "minimal" configuration.

Reply:
Makes sense, removing that from minimal config